docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -95,7 +103,13 @@ If your module does not need to target a remote host, you can quickly and easily
 
     ANSIBLE_LIBRARY=./library ansible -m my_test -a 'name=hello new=true' localhost
 
--  If for any reason (pdb, using print(), faster iteration, etc) you want to avoid going through Ansible,
+For module developed in an existing collection called ``foo.bar`` (As mentioned above):
+
+.. code:: shell
+
+    $ ansible localhost -m foo.bar.my_test -a 'name=hello new=true' --playbook-dir=$PWD
+
+-  If for any reason (``pdb``, using ``print()``, faster iteration and so on) you want to avoid going through Ansible,
```

Comment:
This reads a little strangely to me to be honest. The point is if you want to avoid going through Ansible while developing the module so you can more quickly see issues, right? How about this?

If you use ``pdb``, ``print()``, or some other method of local debugging for faster iteration, you can avoid going through Ansible by creating an arguments file...